crt_portal/cts_forms/templates/partials/attachment-removal-confirmation-modal.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<div id="attachment-removal-confirmation--modal" hidden>
+  <div class="modal-wrapper" role="dialog" aria-modal="true" aria-describedby="modal_dialog_header">
+    <div class="modal-content modal-content--small">
+      <div class="modal-header">
+        <h1 id="modal_dialog_header" class="h2__display">
+          {% trans "Remove Attachment" %}
+        </h1>
+      </div>
+      <div class="modal-form">
+        <p>
+          {% trans "Are you sure you want to remove the following attachment?" %}
+          <h3 id="attachment-removal--filename">
+          </h3>
+        </p>
+        <div class="modal-footer">
+          <a id="attachment-removal--no" href="#" class="usa-button usa-button--outline button--cancel light-button">{% trans "No" %}</a>
```

Comment:
The text here is rendering in Red when the element is active/focused.
<img width="221" alt="Screen Shot 2021-03-15 at 11 24 41 AM" src="https://user-images.githubusercontent.com/3485564/111177920-0ff63080-8581-11eb-96fd-4ac5b448b86b.png">
?

Reply:
made changes to css to fix this.